src/modules/ethereum/transactions/gasPrices/EIP1559.ts

```diff
@@ -1,8 +1,4 @@
-export interface EIP1559 {
-  estimation_seconds?: number;
-  max_fee: number;
-  priority_fee: number;
-}
+import type { EIP1559 } from '@zeriontech/transactions/lib/shared/user-configuration/types';
```

Comment:
should probably re-export in `@zeriontech/tranasactions` index file to avoid such paths